etna/loggers/wandb_logger.py

```diff
@@ -101,10 +101,10 @@ def log(self, msg: Union[str, Dict[str, Any]], **kwargs):
 
         Notes
         -----
-        We log nothing via current method in wandb case.
-        Currently you could call ``wandb.log`` by hand if you need this.
+        We log dictionary to wandb only.
         """
-        pass
+        if isinstance(msg, dict):
```

Comment:
Don't we want to throw warning in the other cases?

Reply:
I guess it would be too noisy in stdeer